accumulator/utils.go

```diff
@@ -338,11 +346,45 @@ func treeRows(n uint64) uint8 {
 
 }
 
+// logicalTreeRows returns the number of
+//
+// Example: The below tree will return 1 as the logical number of rows is 1 for this
+// forest.
+//
+// row 2:
+//        |-------\
+// row 1: 04
+//        |---\   |---\
+// row 0: 00  01  02
+func logicalTreeRows(n uint64) uint8 {
```

Comment:
What's the difference between `treeRows()` and `logicalTreeRows()`?  `treeRows()` is a bit fancier but they do the same thing right?

Reply:
`treeRows()` will return the amount of rows that the forest will allocate for those number of leaves. `logicalTreeRows()` will return only the amount of rows that need to be there.

I put an example here with `numLeaves := uint64(3)`. `treeRows()` returns 2 because the forest will allocate enough for 2^2 leaves. But `logicalTreeRows()` returns 1 because that's where the highest root is.
https://play.golang.org/p/m-EWcl0aw0V

Reply:
How about just using `uint8(bits.Len64(n) - 1)` in the code instead of making a new function? It's a single liner too.

Reply:
OK sure, that works.